Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml

```diff
@@ -117,11 +114,28 @@ script:
     - contextPath: FireMonSecurityManager.SIQL.matchId
       description: Resposne for the SIQL query.
       type: Unknown
-  dockerimage: demisto/python3:3.10.13.88772
+  - arguments:
+    - name: page
+    - name: pageSize
```

Comment:
Please add them argument description

Reply:
The description of these two parameters has been added.